arangodb/content.md

```diff
@@ -65,65 +56,76 @@ These can be used to access the database.
+See `docker inspect --format "{{ .Config.Volumes}}" arangodb/arangodb` for all volumes.
+
+A good explanation about persistence and docker container can be found here:[Docker In-depth: Volumes](http://container42.com/2014/11/03/docker-indepth-volumes/),[Why Docker Data Containers are Good](https://medium.com/@ramangupta/why-docker-data-containers-are-good-589b3c6c749e)
 
 ### Using host directories
 
 You can map the container's volumes to a directory on the host, so that the data is kept between runs of the container. This path `/tmp/arangodb` is in general not the correct place to store you persistent files - it is just an example!
 
-```console
-$ mkdir /tmp/arangodb
-$ docker run -p 8529:8529 -d \
-          -v /tmp/arangodb:/var/lib/arangodb \
-          arangodb
-```
+	unix> mkdir /tmp/arangodb
+	unix> docker -e ARANGO_RANDOM_ROOT_PASSWORD=1 run -p 8529:8529 -d \
+	          -v /tmp/arangodb:/var/lib/arangodb \
+	          arangodb
 
 This will use the `/tmp/arangodb` directory of the host as database directory for ArangoDB inside the container.
 
-## Using a custom ArangoDB configuration file
-
-The ArangoDB startup configuration is specified in the file `/etc/arangodb/arangodb.conf`. If you want to use a customized ArangoDB configuration, you can create your alternative configuration file in a directory on the host machine and then mount that directory location as `/etc/arangodb` inside the `%%REPO%%` container.
+### Using a data container
 
-If `/my/custom/arangod.conf` is the path of your arangodb configuration file, you can start your `%%REPO%%` container like this:
+Alternatively you can create a container holding the data.
 
-```console
-$ docker run --name some-%%REPO%% -v /my/custom:/etc/arangodb -d %%REPO%%:tag
-```
+	unix> docker create --name arangodb-persist arangodb/arangodb true
 
-This will start a new container `some-%%REPO%%` where the ArangoDB instance uses the startup settings from your config file instead of the default one.
+And use this data container in your ArangoDB container.
 
-Note that users on host systems with SELinux enabled may see issues with this. The current workaround is to assign the relevant SELinux policy type to your new config file so that the container will be allowed to mount it:
+	unix> docker run -e ARANGO_RANDOM_ROOT_PASSWORD=1 --volumes-from arangodb-persist -p 8529:8529 arangodb
 
-```console
-$ chcon -Rt svirt_sandbox_file_t /my/custom
-```
+If want to save a few bytes you can alternatively use[tianon/true](https://registry.hub.docker.com/u/tianon/true/) or[progrium/busybox](https://registry.hub.docker.com/u/progrium/busybox/) for creating the volume only containers. Please note that you need to provide the used volumes in this case. For example
 
-### Using a data container
+	unix> docker run -d --name arangodb-persist -v /var/lib/arangodb tianon/true true
 
-Alternatively you can create a container holding the data.
+## Building your own ArangoDB image
```

Comment:
This section feels a little out of place, since this will be on the Docker Hub and not in the git repo.